docs/README.md

```diff
@@ -49,6 +50,7 @@ Following sections are Kubernetes specific. If you are Kubernetes user, use foll
 | AWS EBS CSI Driver \ Kubernetes Version| v1.12 | v1.13 | v1.14 | v1.15 | v1.16 | v1.17 | v1.18 |
```

Comment:
Time to make this 1.18+? :)

Reply:
yeah.

i was considering csompressing the rows and columns but I think tehre is some value in acknowleding each version explicitly...to a point. we might be past that point